source/pages/Import/ImportPhrase.tsx

```diff
@@ -41,6 +48,12 @@ const ImportPhrase: FC<IImportPhrase> = ({ onRegister }) => {
             },
             () => ({
               validator(_, value) {
+                try {
+                  value = formatSeedPhrase(value);
```

Comment:
```suggestion
                  form.setFieldsValue({ phrase: formatSeedPhrase(value) });
```

you can't set the field value directly using the value param